docs/visual-basic/programming-guide/language-features/variables/object-variable-declaration.md

```diff
@@ -113,6 +113,8 @@ End Sub
   
  <xref:System.Object>  
   
+ `` <xref:System.MarshalByRefObject>  
```

Comment:
We also need to reinstate the hierarchy in this section. Perhaps you need to change this section to use `&nbsp;` to force the whitespace.

```markdown
<xref:System.Object> 
&nbsp;&nbsp;<xref:System.MarshalByRefObject> 
&nbsp;&nbsp;&nbsp;&nbsp;<xref:System.ComponentModel.Component>  
etc.
```

which I'm hoping it will render as:
Object
&nbsp;&nbsp;MarshalByRefObject
&nbsp;&nbsp;&nbsp;&nbsp;Component

Makes sense?